tutorials/interoperability/remote-batched-evm-calls.md

```diff
@@ -0,0 +1,230 @@
+0xc10a04630003000100a10f031000040000010403000f0000c16ff28623130000010403000f0000c16ff28623000601070053cd200a007d09260001f0490200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008080000000000000000000000000000000000000000000000000000000000000000110896e292b8000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000e0000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000001e000000000000000000000000000000000000000000000000000000000000000020000000000000000000000001fc56b105c4f0a1a8038c2b429932b122f6b631f000000000000000000000000ed13b028697febd70f34cf9a9e280a8f1e98fd29000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000042004ffd90000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000042004ffd9000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000d0100000103000d1587fae4f265b31ea9ba0cfdb1ade1208ed657
+```
+
+Now that we have the SCALE encoded calldata, the last step is to submit the transaction, which will send our XCM message to Moonbase Alpha, and do the remote batched EVM call!
+
+## Sending the XCM Message from the Relay Chain {: #send-xcm-message-relay-chain }
+
+Congratulations on making it here, you're almost done! Let's recap what we've done so far:
+
+ - We've created a relay chain account that is funded with `UNIT` tokens (relay chain native tokens)
+ - We determined its multilocation-derivative account on Moonbase Alpha and funded this new address with `DEV` tokens
+ - We obtained the Batch precompile calldata which combines two mint calls for `Mars` and `Neptune` ERC-20 tokens.
+ - We built the SCALE encoded calldata in Moonbase Alpha to access its EVM via XCM
+ - We crafted our transaction to send an XCM message to Moonbase Alpha, in which we will ask it to execute the SCALE encoded calldata that was previously built. This, in turn, will execute the call to the batch precompile which includes the mint calls for both the `Mars` and `Neptune` ERC-20 tokens!
+
+To send the XCM message that we built in the previous section, you can use the following code snippet:
+
+```js
+--8<-- 'code/tutorials/interoperability/remote-batched-evm-calls/send-xcm-message.js'
+```
+
+Once you have the code set up, you can execute it with `node`, and the XCM message will be sent to initiate your call to the batch precompile for the mints of `Mars` and `Neptune` ERC-20 tokens in Moonbase Alpha. Don't worry if you see an `Abnormal Closure` error. Rest assured the XCM message was sent successfully. 
```

Comment:
Instead of saying rest assured that it was sent successfully, i think mentioning that they can check their balances to ensure that everything worked as expected